hkmc2/shared/src/test/mlscript-compile/apps/parsing-web-demo/main.mls

```diff
@@ -0,0 +1,223 @@
+import "../parsing/Parser.mls"
+import "../parsing/Lexer.mls"
+import "../../Iter.mls"
+import "../../XML.mls"
+import "../../Option.mls"
+import "../../Runtime.mls"
+import "../../Predef.mls"
+import "../parsing/TreeHelpers.mls"
+import "../parsing/Extension.mls"
+import "../parsing/ParseRuleVisualizer.mls"
+import "../parsing/Rules.mls"
+import "../parsing/vendors/railroad/railroad.mjs"
+
+open XML { elem }
+open Predef
+open Option { Some, None }
+
+declare val document
+declare val customElements
+declare class HTMLElement
```

Comment:
Maybe add these to `Prelude.mls`?

Reply:
We can start by doing it this way. But I would like to eventually have a `WebPrelude.mls` specifically for web, as there will be many things that need to be declared.

Reply:
Yes, it would be better, esp. since other dialects/backends shouldn't have these in their prelude.